docs/examples/here-data-sdk-aaa-example/README.md

```diff
@@ -0,0 +1,42 @@
+# Simple Authorization WEB App 
+
+This is a simple Web App with configured compiling TS to JS and bundling with webpack.
+The app shows how to use APIs from `@here/olp-sdk-dataservice-api`; 
+
+All function from AuthorizationAPI can be used in the same way as used in this example.
+
+## Setup
+
+1) Download the current folder to your PC.
+2) Add to your enviropment app key and secret.
+```
+export OLP_APP_KEY=<your app key>
+export OLP_APP_SECRET=<your app secret>
+```
+3) Install dependencies.
+
+```
+npm install
+```
```

Comment:
```suggestion
## Set up

1. Download the current folder to your PC.
2. Add your app key and secret to your enviropment.
    ```
    export OLP_APP_KEY=<your app key>
    export OLP_APP_SECRET=<your app secret>
    ```
3. Install the dependencies.

    ```
    npm install
    ```
```

Reply:
Set up - ignored, Setup should be used.